src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -215,12 +220,33 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
 
             internal static async Task<IReadOnlyDictionary<string, object>> GetRequestBindingDataAsync(HttpRequestMessage request, Dictionary<string, Type> bindingDataContract = null)
             {
+                var requestBinding = new JObject();
```

Comment:
We need to think how this relates to the existing request data we make available in functions. Currently on the context object we have:

```javascript
context.req
context.req.headers
context.req.body
context.req.originalUrl
context.req.method
context.query
```

We are translating the request in to an IDictionary<string, object> already for context. This is just not part of binding data. I'm wondering if we should move some of this logic to the http binding...